modules/instance_template/outputs.tf

```diff
@@ -23,3 +23,8 @@ output "name" {
   description = "Name of instance template"
   value       = google_compute_instance_template.tpl.name
 }
+
+output "target_tags" {
+  description = "Tags that will be associated with instance(s)"
+  value       = var.tags
```

Comment:
We should expose these from the template, rather than passing through the var directly. (This ensures they can work properly in the dependency graph.)